reference/carts.sf.yml

```diff
@@ -537,7 +537,7 @@ components:
                 $ref: '#/components/schemas/requestCartPostLineItem'
             version:
               type: integer
-              description: The expected version of the cart.
+              description: The cart version to which updates are expected to apply. If the provided version doesn't match the current cart version, a conflict error will be returned. This field is optional; if not provided, optimistic concurrency control will not apply.
```

Comment:
```suggestion
              description: The cart version that you expect to apply the updates. If the provided version doesn't match the current cart version, you will receive a conflict error. This field is optional; if not provided, optimistic concurrency control will not apply.
```

Reply:
I will update the other occurrences. @bc-andreadao